### PR TITLE
Adding Documentation for AzureADAdministrators

### PR DESCRIPTION
**Pull request by jaczhan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Hi @jaczhan there are some build failures which might be due to the strange white-space characters in ServerAdminCreateUpdate.json and ServerAdminGet.json (see the red blocks in the diff).  Can you please clean these up?

**Comment by jaczhan** [comment]
@jhendrixMSFT

Hi, sorry, not really familiar with git so I just edit and recommitted those two files online. Anything else needed to re-run the build ? Thanks.




Jack

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators/activedirectory":{  
+         "put":{  
+            "tags":[  
+               "ServerAdministrators"
+            ],
+            "operationId":"ServerAzureADAdministrators_CreateOrUpdateAzureADAdministrator",
+            "description":"Creates a new Server Active Directory Administrator or updates an existing server Active Directory Administrator.",
+            "x-ms-examples":{  
+               "Create/Update a server administrator":{  
+                  "ref":"../examples/ServerAdminCreateUpdate.json"
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: This is an invalid ref, it should be "$ref".

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+         },
+         "delete":{  
+            "tags":[  
+               "ServerAdministrators"
+            ],
+            "operationId":"ServerAzureADAdministrators_Delete",
+            "description":"Deletes an existing server Active Directory Administrator.",
+            "x-ms-examples":{  
+               "Delete a server administrator":{  
+                  "ref":"../examples/ServerAdminDelete.json"
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: Should be $ref.

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+         },
+         "get":{  
+            "tags":[  
+               "ServerAdministrators"
+            ],
+            "operationId":"ServerAzureADAdministrators_Get",
+            "description":"Returns an server Administrator.",
+            "x-ms-examples":{  
+               "Get the Active Directory Administrator":{  
+                  "ref":"../examples/ServerAdminGet.json"
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: Should be $ref.

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators":{  
+         "get":{  
+            "tags":[  
+               "ServerAdministrators"
+            ],
+            "operationId":"ServerAzureADAdministrators_List",
+            "description":"Returns a list of server Administrators.",
+			"x-ms-examples":{  
+               "get a list of server administrators":{  
+                  "ref":"../examples/ServerAdminList.json"
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: Should be $ref.

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+   "produces":[  
+      "application/json"
+   ],
+   "paths":{  
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators/activedirectory":{  
+         "put":{  
+            "tags":[  
+               "ServerAdministrators"
+            ],
+            "operationId":"ServerAzureADAdministrators_CreateOrUpdateAzureADAdministrator",
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: This operation ID should be ServerAzureADAdministrators_CreateOrUpdate.

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+      "https"
+   ],
+   "consumes":[  
+      "application/json"
+   ],
+   "produces":[  
+      "application/json"
+   ],
+   "paths":{  
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators/activedirectory":{
```
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: This path looks wrong to me, having the fixed "activedirectory" at the end.  Shouldn't it be something like "{administratorName}"?  @ravbhatnagar is this ok per ARM rules?

**Comment by jhendrixMSFT** [comment]
@jaczhan Please have a look at the model validation errors, there are some discrepancies between the swagger and examples.
https://travis-ci.org/Azure/azure-rest-api-specs/jobs/230554697

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json (hunk 6 above)
Comment (jaczhan) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: There is only one aad administrator is allow( a person or a group), so the "{administratorName}" in the api is a fix string "activedirectory" always. confirmed with .Net test

File: arm-sql/compositeSql.json [hunk 7]
```diff
@@ -12,6 +12,7 @@
     "./2014-04-01/swagger/replicationLinks.json",
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
+	"./2014-04-01/swagger/serverAzureADAdministrators.json",
```
Comment (jhendrixMSFT) [inline arm-sql/compositeSql.json]: Nit, spacing is off.

File: arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json (hunk 6 above)
Comment (jhendrixMSFT) [inline arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json]: I see thanks for clarifying.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**